dialogue-core/src/main/java/com/palantir/dialogue/core/FixedLimitedChannel.java

```diff
@@ -51,6 +54,7 @@
         boolean optimisticallyAcquiredPermit = usedPermits.incrementAndGet() > totalPermits;
         if (optimisticallyAcquiredPermit) {
             returnPermit.run();
+            limitedMeter.mark();
```

Comment:
Should these be tagged by endpoint / service ? I know it would be expensive, but I'm imagining looking at a datadog graph and seeing a big chunk of limits, but not knowing whether they're evenly spread across all outgoing requests or just a few of them?

Reply:
I'm not sure we should add endpoint data per #407. Service might be alright, for now I'd like to record some data so we can figure out where things are getting flagged.